test/fixtures/output/kafka/topic/update-success-rest.golden

```diff
@@ -1,4 +1,4 @@
-Updated the following configuration values for topic "topic-exist-rest" (read-only configs were not updated):
+Updated the following configuration values for topic "topic-exist-rest":
         Name       | Value | Read-Only  
 -------------------+-------+------------
   compression.type | gzip  | false      
```

Comment:
I'm considering hiding entire columns in the human output if they're marked as `omitempty` and every row contains a default value. What do you think?

Reply:
i think it still make sense for users to see in the output, which / those changes were ineffective